providers/implementations/encode_decode/decode_der2key.c

```diff
@@ -136,6 +138,8 @@ static void *der2key_decode_p8(const unsigned char **input_der,
             ctx->flag_fatal = 1;
         X509_SIG_free(p8);
     } else {
+        // Pop any errors that might have been raised by d2i_X509_SIG.
```

Comment:
Use `/**/`

Reply:
Sorry about that, I've updated now. Thanks